tests/nemo_text_processing/zh/data_text_normalization/test_cases_cardinal.txt

```diff
@@ -138,7 +138,6 @@
 100000100000~一千亿零十万
 100000010000~一千亿零一万
 100000001000~一千亿零一千

```

Comment:
@BuyuanCui could you please add sentence-level test cases with semiotic classes in context? at least a few for every semiotic class. Thanks!

Reply:
Would add sentences to MEASURE and WORD.